### DISCUSSION TITLE
The submitted file example only released in PaddlePaddle version, so do we need to write own submit file if we use other framework?

### DISCUSSION THREAD

**Discussion by GuibinChen92057** [PaddlePaddle/PaddleSpatial#249] [BaiduKDDCup2022] [answered]
The released submitted file contains the predict.py and test_data.py files is the PaddlePaddle version, so if we use other framework like pytorch, we need to write all these .py files from scratch? 
![image](https://user-images.githubusercontent.com/85916561/178149740-20a0af65-b1cc-454b-b90e-fa2e48190cd7.png)

**Comment by ramber1836** [comment, marked as answer]
No, not necessary.

**Comment by GuibinChen92057** [comment]
thx! I will try to use released version



---- Replied Message ----
| From | Xinjiang ***@***.***> |
| Date | 07/11/2022 10:12 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [PaddlePaddle/PaddleSpatial] The submitted file example only released in PaddlePaddle version, so do we need to write own submit file if we use other framework? (Discussion #249) |

No, not necessary.
